sys/conf/files.arm64

```diff
@@ -47,6 +47,7 @@ arm64/arm64/debug_monitor.c			standard
 arm64/arm64/disassem.c				optional ddb
 arm64/arm64/dump_machdep.c			standard
 arm64/arm64/efirt_machdep.c			optional efirt
+arm64/arm64/efirt_support.S			optional efirt
```

Comment:
I think you can make this `optional efirt cheri_purecap_kernel` and then remove the #ifdef's in the source file.

Reply:
Hm, I chose not to do that because this is where the non-nofault version would live (that's the name of the file that contains it on amd64)

Reply:
Ok.